src/serverenvironment.cpp

```diff
@@ -1015,11 +1020,12 @@ void ServerEnvironment::activateBlock(MapBlock *block, u32 additional_dtime)
 	activateObjects(block, dtime_s);
 
 	/* Handle LoadingBlockModifiers */
-	m_lbm_mgr.applyLBMs(this, block, stamp, (float)dtime_s);
+	if (block->getParent()) // Block not deleted
```

Comment:
this check should be done in apply directly no ? to prevent coding errors while calling apply

Reply:
`applyLBMs` takes a valid block. One should imo stop using a block for anything as early as possible, and not pass it around.

FYI, this check is here because of `activateObjects`. The given block should be guaranteed not to be orphan before that call.
(I've just noticed that some orphan checks were missing in `activateObjects` (i.e. after `on_activate` or metatable functions), I've added them.)